pyclesperanto/_categories.py

```diff
@@ -0,0 +1,17 @@
+def categories():
+
+    from inspect import getmembers, isfunction
+
+    import pyclesperanto as cle
```

Comment:
Is this the only way? not a big fan of having an `import` of the package inside the package ...